src/java.base/share/classes/java/lang/foreign/SymbolLookup.java

```diff
@@ -138,10 +139,12 @@ public interface SymbolLookup {
      * this method returned.
      * <p>
      * Libraries associated with a class loader are unloaded when the class loader becomes
-     * <a href="../../../java/lang/ref/package.html#reachability">unreachable</a>. The symbol lookup
-     * returned by this method is backed by a {@linkplain MemorySession#asNonCloseable() non-closeable}, shared memory
-     * session which keeps the caller's class loader reachable. Therefore, libraries associated with the caller's class
-     * loader are kept loaded (and their symbols available) as long as a loader lookup for that class loader is reachable.
+     * <a href="../../../java/lang/ref/package.html#reachability">unreachable</a>. The returned loader lookup
+     * keeps the caller's class loader reachable. Therefore, libraries associated with the caller's class
+     * loader are kept loaded (and their symbols available) as long as a loader lookup for that class loader
+     * (or any of the symbols originated from it) is reachable.
+     * <p>
+     * The symbols obtained from the returned loader lookup are backed by the {@linkplain MemorySession#global() global session}.
      * <p>
```

Comment:
Does the spec update mean this should have a CSR?

Reply:
Yep - note that PR is in draft mode as we need to think through what changes we actually want to pursue.